src/utils.js

```diff
@@ -187,7 +230,7 @@ const FOCUSABLE_SELECTOR = [
 function isFocusable(element) {
   return (
     !isLabelWithInternallyDisabledControl(element) &&
-    element?.matches(FOCUSABLE_SELECTOR)
+    (isContentEditable(element) || element?.matches(FOCUSABLE_SELECTOR))
```

Comment:
Actually, could we change this to update the `FOCUSABLE_SELECTOR` variable to include `[contenteditable=""]` and `[contenteditable="true"]`. Then we don't need to make any changes here.

Reply:
The benefit of this is that then the `FOCUSABLE_SELECTOR` will really be able to tell us whether something is focusable (with the exception of a label with internally disabled control... I wonder whether that would be possible to handle in the selector as well).